test/sql-luatest/sql_func_expr_test.lua

```diff
@@ -0,0 +1,66 @@
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+local g = t.group()
+
+g.before_all(function()
+    g.server = server:new({alias = 'sql_func_expr'})
+    g.server:start()
+end)
+
+g.after_all(function()
+    g.server:stop()
+end)
+
+-- Make sure CHECK constraint works as intended.
+g.test_sql_func_expr_1 = function()
+    g.server:exec(function()
+        local t = require('luatest')
+        local def = {language = 'SQL_EXPR', is_deterministic = true,
+                     body = 'a * b > 10'}
+        box.schema.func.create('abc', def)
```

Comment:
I don't see any tests that check recovery of the new type of stored function from xlog and snapshot.

Reply:
Fixed, added a new test-case.